fact-bounty-flask/api/commands.py

```diff
@@ -168,3 +191,34 @@ def deploy():
     migrate database to latest revision
     """
     upgrade()
+
+
+@click.command(name="create_admin")
+@with_appcontext
+def create_admin():
+    """
+    create an admin user
+    """
+    admin_username = input("Enter the admin username: ")
+    admin_password = getpass.getpass(prompt="Enter the admin password: ")
+    admin_role = "admin"
+    valid = False
+    while not valid:
+        admin_email = input("Enter the admin email: ")
+        valid = check(admin_email)
+        if valid:
+            break
+        else:
+            pass
+    try:
+        user = model.User(
+            name=admin_username,
+            email=admin_email,
+            password=admin_password,
+            role=admin_role,
+        )
+        user.save()
+    except Exception as e:
+        print(str(e))
+        response = {"message": "Something went wrong!!"}
+        return make_response(jsonify(response)), 500
```

Comment:
It's unclear why this method return a response like a route, it's just a click command. Also please attach some screenshot demonstrating this change.